smoker/client/__init__.py

```diff
@@ -19,16 +19,20 @@ class Client(object):
     """
     hosts = []
 
-    def __init__(self, hosts):
+    def __init__(self, hosts, config):
```

Comment:
how about passing only the port number, together with the default parameter value?
I mean something like:
```python
def __init__(self, hosts, port=8086)
```